src/lib/accessibility/Behaviors/List/BasicListItemBehavior.ts

```diff
@@ -6,12 +7,31 @@ import { IAccessibilityDefinition } from '../../interfaces'
  * The 'listitem' role is used to identify an element that is a single item in a list.
  */
 
-const BasicListItemBehavior: IAccessibilityDefinition = {
```

Comment:
please add keyboard handling o selectable list only - simple list is not navigable/focusable

Reply:
fixed